src/app/machines/views/MachineDetails/MachineNetwork/InterfaceFormTable/InterfaceFormTable.test.tsx

```diff
@@ -88,17 +76,17 @@ describe("InterfaceFormTable", () => {
         system_id: "abc123",
       }),
     ];
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <InterfaceFormTable
-          interfaces={[{ nicId: nic.id }]}
-          selectedEditable
-          systemId="abc123"
-        />
-      </Provider>
+    renderWithMockStore(
+      <InterfaceFormTable
+        interfaces={[{ nicId: nic.id }]}
+        selectedEditable
+        setSelected={jest.fn()}
+        systemId="abc123"
+      />,
+      { state }
     );
-    expect(wrapper.find("NameColumn").prop("showCheckbox")).toBe(true);
+
+    expect(screen.getByRole("checkbox")).toBeInTheDocument();
```

Comment:
This is less explicit than the original implementation. Could you provide a checkbox label?